TODO.md

```diff
@@ -0,0 +1,8 @@
+To Do
+-----
+
+  * Load all frames from TIFF files. We have to write our own loader for this to
+    happen--just like we did for GIF images--because Imlib2 does not support
+    multiple frames. Issue #241.
+  * Add support for more embedded thumbnail formats. Right now, nsxiv seems to use
+    the smallest one. Issue #238.
```

Comment:
If we're going markdown, might as well use the fancy todo list feature.

```suggestion
- [ ] Load all frames from TIFF files. We have to write our own loader for this to
    happen--just like we did for GIF images--because Imlib2 does not support
    multiple frames. Issue #241.
- [ ] Add support for more embedded thumbnail formats. Right now, nsxiv seems to use
    the smallest one. Issue #238.
```

Reply:
Ahh, also realized. `Issue #241.`,  `Issue #241` these won't work. We'll need to hardlink to sxiv's repo.

Reply:
Wouldn't we remove the entry after it's done? I don't think `- [ ]` is necessary here.